[2023-08-30T01:43:48.376+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: vnstock-pipeline-1H.Vm_pub_pubsub scheduled__2023-08-29T09:45:00+00:00 [queued]>
[2023-08-30T01:43:48.382+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: vnstock-pipeline-1H.Vm_pub_pubsub scheduled__2023-08-29T09:45:00+00:00 [queued]>
[2023-08-30T01:43:48.382+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2023-08-30T01:43:48.395+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Vm_pub_pubsub> on 2023-08-29 09:45:00+00:00
[2023-08-30T01:43:48.398+0000] {standard_task_runner.py:57} INFO - Started process 220405 to run task
[2023-08-30T01:43:48.399+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2023-08-30T01:43:48.405+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'vnstock-pipeline-1H', 'Vm_pub_pubsub', 'scheduled__2023-08-29T09:45:00+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpal1tvexi']
[2023-08-30T01:43:48.406+0000] {standard_task_runner.py:85} INFO - Job 176: Subtask Vm_pub_pubsub
[2023-08-30T01:43:48.443+0000] {task_command.py:415} INFO - Running <TaskInstance: vnstock-pipeline-1H.Vm_pub_pubsub scheduled__2023-08-29T09:45:00+00:00 [running]> on host vnstock-pipeline-vm.us-central1-c.c.vnstock-pipeline.internal
[2023-08-30T01:43:48.500+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='nguyentuanduong555@gmail.com' AIRFLOW_CTX_DAG_OWNER='Duong' AIRFLOW_CTX_DAG_ID='vnstock-pipeline-1H' AIRFLOW_CTX_TASK_ID='Vm_pub_pubsub' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T09:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T09:45:00+00:00'
[2023-08-30T01:43:49.771+0000] {logging_mixin.py:151} WARNING - /home/nguyentuanduong7/airflow/src/pluggin/stock_subscription.py:21 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2023-08-30T01:43:49.783+0000] {logging_mixin.py:151} INFO -   Mã CP  Giá tham chiếu  Giá Trần  Giá Sàn  Giá khớp lệnh  Tổng Khối Lượng                              time
0   ACB         22050.0   23550.0  20550.0            0.0              0.0  2023-08-30 08:43:34.211957+07:00
1   TCB         33900.0   36250.0  31550.0            0.0              0.0  2023-08-30 08:43:34.211957+07:00
[2023-08-30T01:43:49.887+0000] {logging_mixin.py:151} INFO - publish message id 9406311282080068
[2023-08-30T01:43:49.946+0000] {logging_mixin.py:151} INFO - publish message id 8312154737876550
[2023-08-30T01:43:49.946+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-08-30T01:43:49.950+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=vnstock-pipeline-1H, task_id=Vm_pub_pubsub, execution_date=20230829T094500, start_date=20230830T014348, end_date=20230830T014349
[2023-08-30T01:43:49.978+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T01:43:49.989+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
